npm start?

In `getting_started` tutorial there is the following paragraph

```
Start up your server with npm start and go to http://localhost:3000/hello in your browser. 
Oh no! We're getting a 404 error because we never created a hello.html file. 
You need to create the missing file to get rid of this error.
```

When I enter `npm start` in the command line I get the following

```
npm ERR! Linux 4.2.0-23-generic
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "start"
npm ERR! node v5.2.0
npm ERR! npm  v3.3.12

npm ERR! missing script: start
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /home/tom/hacking/learnjs/npm-debug.log
```

The homepage recommends "Launch the application by running `node .`" which works for me. So I'm confused about the alternate way to start the application via the node package manager.

--- a/templates/index.jade
+++ b/templates/index.jade
@@ -58,7 +58,7 @@ block content
       .highlight.highlight-bash
         pre npm install hapi --save
 
-      p Create an 
+      p Create a
         code server.js
         |  file with the following contents:
 

--- a/templates/index.jade
+++ b/templates/index.jade
@@ -59,7 +59,7 @@ block content
         pre npm install hapi --save
 
       p Create an 
-        code index.js
+        code server.js
         |  file with the following contents:
 
       .highlight.highlight-javascript
@@ -173,7 +173,7 @@ block content
           br
 
         p Launch the application by running 
-          code node .
+          code npm start
           |  and open 
           em localhost:8000/hello
           |  in your browser.

